web-ng/src/app/components/main-page/main-page.component.ts

```diff
@@ -70,6 +75,14 @@ export class MainPageComponent implements OnInit {
         .getObservationId$()
         .subscribe(id => this.editObservation(id))
     );
+    // Redirect to sign in page if user is not authenticated.
+    this.subscription.add(
+      this.authService.isAuthenticated$().subscribe(isAuthenticated => {
+        if (!isAuthenticated) {
+          this.router.navigate([AuthGuard.SIGN_IN_URL]);
```

Comment:
Would it be possible to move the this constant into AuthService so we don't need to introduce a dep on AuthGuard in component? Ideally components will be entirely independent of lower level details and global effects like auth guards.